text/refactor_refman.md

```diff
@@ -112,7 +178,11 @@ https://github.com/coq/coq/pull/10614, for example, updates the
 
 # Disadvantages
 
-
+Working on refactoring the manual incrementally will create
+intermediate states which can be more confusing than what we started
+with.  If we end up not having finalized the refactoring towards the
+end of a release cycle, we will need to make adjustments so that it
+still looks good.
 
```

Comment:
>> Could be more positive:

>>  Improving the documentation incrementally may require additional effort to ensure that the documentation remains usable.

> Well, I don't want to diminish the drawbacks.

I think the first sentence is too strong.  Many of the changes can be done without hurting usability.

> Working on refactoring the manual incrementally will create
intermediate states which can be more confusing than what we started
with.